consensus/state.go

```diff
@@ -1486,7 +1473,9 @@ func (cs *State) enterPrecommit(height int64, round int32) {
 		return
 	}
 
-	// If +2/3 prevoted for proposal block, stage and precommit it
+	// If greater than 2/3 of the voting power on the network prevoted for
```

Comment:
The previous if case is just a sub-case of the following if, and should be coded as such. It will be handled in a follow up PR.